api/src/main/java/org/pmiops/workbench/utils/mappers/WorkspaceMapper.java

```diff
@@ -241,4 +242,24 @@ default String cdrVersionId(CdrVersion cdrVersion) {
 
   @Mapping(target = "terraName", source = "workspace.name")
   TestUserRawlsWorkspace toTestUserRawlsWorkspace(RawlsWorkspaceDetails workspace, String username);
+
+  @Mapping(target = "workspaceId", source = "workspace.id")
+  @Mapping(target = "namespace", source = "workspace.userFacingId")
```

Comment:
are those mapped correct? 
(1) workspace.id is string or integer? https://github.com/verily-src/terra-service-ws-manager/blob/3ace6980aa60f6a48658757659e9565ff20b6a0c/service/src/main/java/bio/terra/workspace/service/workspace/model/Workspace.java#L137
AoU workspace id is integer while WSM uses string? 
(2) namespace is currently used in workspace URL from UI. Feels it should be mapped to workspaceId instead?

Reply:
remember that here I map the wsm WorkspaceDescription to RawlsWorkspaceDetails to keep createWorkspace contract consistent with the existing code. workspaceId is String in both models.
About the namespace, I confirmed that both the id and userFacingId are returned as identical, but I agree I should modify it to id to make it more explicit.

Reply:
![image](https://github.com/user-attachments/assets/75f275a8-4807-48ff-8f99-788a3c9c2142)